calendar.go

```diff
@@ -476,7 +476,7 @@ func (cs *CalendarStream) ReadLine() (*ContentLine, error) {
 	for c {
 		var b []byte
 		b, err = cs.b.ReadBytes('\n')
-		if b == nil || len(b) == 0 {
+		if len(b) == 0 {
```

Comment:
I guess that's fair.. I like always checking for null. :P

Reply:
I can also remove that commit. But `len(b)` wil always be `0` if `b == nil`. So it's not really needed, right?
https://play.golang.org/p/5bjyfO5OmB

Reply:
Oh no. No need. You're perfectly correct. I just like to write in a way which assumes as little as possible.